diff --git a/tests/test_pyav.py b/tests/test_pyav.py
index ec00020..618d143 100644
--- a/tests/test_pyav.py
+++ b/tests/test_pyav.py
@@ -364,8 +364,8 @@ def test_bayer_write():
 
 def test_sequential_reading(test_images):
     expected_imgs = [
-        iio.imread(test_images / "cockatoo.mp4", index=1),
-        iio.imread(test_images / "cockatoo.mp4", index=5),
+        iio.imread(test_images / "cockatoo.mp4", index=1, plugin="pyav"),
+        iio.imread(test_images / "cockatoo.mp4", index=5, plugin="pyav"),
     ]
 
     with iio.imopen(test_images / "cockatoo.mp4", "r", plugin="pyav") as img_file:
