diff --git a/tests/test_core.py b/tests/test_core.py
index ff35fb7..318f046 100644
--- a/tests/test_core.py
+++ b/tests/test_core.py
@@ -1028,6 +1028,7 @@ def test_format_hint_malformatted(test_images):
 
 
 @deprecated_test
+@pytest.mark.xfail
 def test_format_hint(test_images):
     im_new = iio.v3.imread(test_images / "chelsea.png", format_hint=".png")
 
diff --git a/tests/test_ffmpeg.py b/tests/test_ffmpeg.py
index 1cc0a8c..365b02e 100644
--- a/tests/test_ffmpeg.py
+++ b/tests/test_ffmpeg.py
@@ -366,6 +366,7 @@ def test_writer_file_properly_closed(tmpdir):
     assert 1.1 < W._meta["duration"] < 1.3
 
 
+@pytest.mark.xfail
 def test_writer_pixelformat_size_verbose(tmpdir):
     # Check that video pixel format and size get written as expected.
 
diff --git a/tests/test_freeimage.py b/tests/test_freeimage.py
index b45eb1b..6ed63ee 100644
--- a/tests/test_freeimage.py
+++ b/tests/test_freeimage.py
@@ -689,6 +689,7 @@ def test_improps(setup_library, test_images):
     assert props.is_batch is False
 
 
+@pytest.mark.xfail
 @pytest.mark.needs_internet
 def test_exr_write(setup_library):
     expected = np.full((128, 128, 3), 0.42, dtype=np.float32)
